api/package.json

```diff
@@ -12,8 +12,9 @@
     "migrate-seed": "knex migrate:latest && knex seed:run",
     "generate-keys": "node src/bin/generate-keys.js",
     "migrate": "knex migrate:latest",
+    "test:prepare": "npm run create-test-db && npm run rebuild-db-for-tests",
     "test": "NODE_ENV=test knex migrate:latest && lab -lv",
-    "test:watch": "NODE_ENV=test knex migrate:latest && nodemon --exec lab -lv",
+    "test:watch": "NODE_ENV=test knex migrate:latest && nodemon --exec lab -lvk",
```

Comment:
I found myself manually doing the steps in `test:prepare` often enough that looking them up annoyed me.

Lab's `-k` flag in `test:watch` hides skipped tests, which _dramatically_ improves output when focused on a single test with `.only()`.